diff --git a/dist/shaka-player.compiled.d.ts b/dist/shaka-player.compiled.d.ts
index a2248b6a1c52bd014a561d66603169dd147771b6..844664e216411a16c88378102465f52930bec034 100644
--- a/dist/shaka-player.compiled.d.ts
+++ b/dist/shaka-player.compiled.d.ts
@@ -5156,3 +5156,4 @@ declare namespace shaka.extern {
 declare namespace shaka.extern {
   type TransmuxerPlugin = ( ) => shaka.extern.Transmuxer ;
 }
+export default shaka;
diff --git a/dist/shaka-player.compiled.debug.d.ts b/dist/shaka-player.compiled.debug.d.ts
index b829443e0271399b8b3546865cb5c15d1b1f3278..79fbbe6b2408e6a383becb425831c3e01c01d21f 100644
--- a/dist/shaka-player.compiled.debug.d.ts
+++ b/dist/shaka-player.compiled.debug.d.ts
@@ -5156,3 +5156,4 @@ declare namespace shaka.extern {
 declare namespace shaka.extern {
   type TransmuxerPlugin = ( ) => shaka.extern.Transmuxer ;
 }
+export default shaka;
diff --git a/dist/shaka-player.ui.d.ts b/dist/shaka-player.ui.d.ts
index 4fdbd43526959a46bac6e7d8c382e99d9449d4ee..2c375703698784f3db52c6c628222d6cefccb3db 100644
--- a/dist/shaka-player.ui.d.ts
+++ b/dist/shaka-player.ui.d.ts
@@ -5869,3 +5869,4 @@ declare namespace shaka.extern {
 declare namespace shaka.extern {
   type UIVolumeBarColors = { base : string , level : string } ;
 }
+export default shaka;
diff --git a/dist/shaka-player.ui.debug.d.ts b/dist/shaka-player.ui.debug.d.ts
index 64121519fc497b6b75d2898828613411c3d30952..191dc6a5390475361aa7ebd0c53da15bb930ff75 100644
--- a/dist/shaka-player.ui.debug.d.ts
+++ b/dist/shaka-player.ui.debug.d.ts
@@ -5869,3 +5869,4 @@ declare namespace shaka.extern {
 declare namespace shaka.extern {
   type UIVolumeBarColors = { base : string , level : string } ;
 }
+export default shaka;
